easybuild/easyblocks/generic/systemcompiler.py

```diff
@@ -168,3 +255,28 @@ def make_module_extend_modpath(self):
         # Reset to actual compiler version (e.g., "4.8.2")
         self.cfg['version'] = self.compiler_version
         return res
+
+    def make_module_extra(self):
```

Comment:
`make_module_extra` takes optional arguments (see https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/framework/easyblock.py#L982), so you need to make sure you pass those down:

```python
def make_module_extra(self, *args, **kwargs):
    ...
        extras = EB_GCC.make_module_extra(self, *args, **kwargs)
    ...
        extras = super(SystemCompiler, self).make_module_extra(*args, **kwargs)